README.md

```diff
@@ -1 +1,54 @@
-# Ride-My-WayApiV1 
+[![Build Status](https://travis-ci.org/TheSteelGuy/Ride-My-WayApiV1.svg?branch=ft-Getmethods-%23158520402)](https://travis-ci.org/TheSteelGuy/Ride-My-WayApiV1)
+[![Coverage Status](https://coveralls.io/repos/github/TheSteelGuy/Ride-My-WayApiV1/badge.svg?branch=master)](https://coveralls.io/github/TheSteelGuy/Ride-My-WayApiV1?branch=master)
+
+# Ride-my-wayAPIv1
+
+
+### Introduction
+Ride-my App is a carpooling application that provides drivers with the ability to create ride offers
+and passengers to join available ride offers.
+Required Features
+1. Users can create an account and log in.
+2. Drivers can add ride offers..
+3. Passengers can view all available ride offers.
+4. Passengers can see the details of a ride offer and request to join the ride. E.g What time
+the ride leaves, where it is headed e.t.c
+5. Drivers can view the requests to the ride offer they created.
+6. Drivers can either accept or reject a ride request.
+
+#### Getting Started
+
+
+
+
+#### Setting
+* First install the virtual environment globally `sudo pip instal virtualenv`
+* create the virtual enviroment `virtualenv --python=python2 myenv`
+* change directory to myenv
+* activate virtual environment `source myenv/bin/activate`
+* clone the repo by running on terminal `git clone https://github.com/TheSteelGuy/Ride-My-WayApiV1.git `
+* run pip install -r requirements.txt
+* change directory to the repo `cd the cloned repo`
```

Comment:
You can use an actual terminal command instead of *cd to the cloned repo*
Since git cloning will clone it to the user's current dir, you can say `cd Ride-My-WayApiV1`